client/src/index.css

```diff
@@ -1,13 +1,7 @@
+@import url('https://fonts.googleapis.com/css?family=Merriweather|Merriweather+Sans:300,400,400i,700,800&display=swap');
 body {
   margin: 0;
```

Comment:
I would remove this margin styling. And would add a spacing attribute to the <Main> Component

Reply:
* to the `Main` component